src/hash/rpo/tests.rs

```diff
@@ -53,37 +53,31 @@ fn hash_elements_vs_merge() {
 }
 
 #[test]
-fn hash_elements_vs_merge_in_domain() {
+fn merge_vs_merge_in_domain() {
     let elements = [Felt::new(rand_value()); 8];
 
     let digests: [RpoDigest; 2] = [
         RpoDigest::new(elements[..4].try_into().unwrap()),
         RpoDigest::new(elements[4..].try_into().unwrap()),
     ];
 
-    // pick a random domain value.
-    let domain = Felt::new(rand_value());
+    // ------------- merge with domain = 0 ----------------------------------------------------------
 
-    // convert the elements into a list of base field elements
-    let elements = Felt::as_base_elements(&elements);
+    let merge_result = Rpo256::merge(&digests);
```

Comment:
tiny nit: since this is used in both tests, I'd probably lift it up above the header to right below the digests declaration, but feel free to leave it as it is if you disagree

Reply:
done